### PR TITLE
Required questions

### PR DESCRIPTION
**Pull request by plusjeff**
For #80 and #81 : All questions asked, except remarks, are required fields.

If a user does not answer a question, they get prompted for it after clicking *Continue*.

image

Remarks are labelled as optional:

image

### CODE REVIEW

**Comment by plusjeff** [comment]
@KHarshawat : Could you review for design, then assign to @alex for code review?

File: app/views/web/questions.html.erb [hunk 1]
```diff
@@ -30,7 +31,8 @@
           <%= render partial: 'form8_yes_no_radio', locals: {
                   question: 'THE AGENT DESIGNATED, IS HE/SHE ON ACCREDITED LIST',
                   question_num: '8C',
-                  input_name: '8C_IF_AGENT_DESIGNATED_IS_HE_SHE_ON_ACCREDITED_LIST'
+                  input_name: '8C_IF_AGENT_DESIGNATED_IS_HE_SHE_ON_ACCREDITED_LIST',
+                  required: true
```
Comment (alex) [inline app/views/web/questions.html.erb]: tiny nit, can you use trailing commas here and in the other ones, saves us from needing to change two lines when we add the next argument.

**Comment by KHarshawat** [comment]
image

**Comment by webinista** [comment]
This also affects #156 and #152.

**Comment by plusjeff** [comment]
@KHarshawat @webinista : Will close this one and wait until #151 is merged in to complete.

**Pull request closed without merging**